backend/gatekeeper/views.py

```diff
@@ -186,8 +159,227 @@ def create(self, request, sts_response):
                 if entity_create_function:
                     entity_create_function(created_object_data)
 
+    def expand_datastreams(self):
+        """
+        Expand the requested url to include datastreams
+
+        As results are filtered by datastreams we want to
+        gain access to all datastreams associated with the
+        results. We do this by expanding the request to
+        include datastreams.
+        """
+
+        params = self.sts_arguments.get('params', {})
+        expand_params = params.get('$expand', '')
+
+        # If we are already extending Datastreams there is no need for further actions
+        if expand_params and self.sts_datastreams_extend_re.match(expand_params):
+            return False
+
+        parse_result = parse_sta_url(self.sts_url, prefix=get_gatekeeper_sta_prefix())
+
+        entity_type = parse_result['parts'][-1]['name']
+        print("Entity type: ", entity_type)
```

Comment:
Print debug

Reply:
Good catch, did not know that flake8-print wasn't added to the project so I added that as well 😄